roles/redhat-subscriptions/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for redhat-subscriptions
+rhsm_username: user@redhat.com
+rhsm_password: QMNBjdkfIJHDkka
```

Comment:
I assume this is fake? Maybe make it something that reads slightly better that way? Like `fake-password`

Reply:
Yes, it fake. Just want to mention this explicitly.